environments/common/inventory/groups

```diff
@@ -202,6 +202,9 @@ k3s_agent
 [extra_packages]
 # Hosts to install specified additional packages on
 
+[extra_packages:children]
```

Comment:
This block is redundant, see above, there should be no change to this file in this PR.